content/guides/tina-cloud/starter/structure.md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
Is this page needed for the guide? Makes sense in the readme but not sure if it adds value to the guide.

Reply:
I thought it would make sense to just have a bit of an explanation. I think we're aiming to have the same-ish set of instructions in both places